templates/case-study/index.html

```diff
@@ -0,0 +1,28 @@
+{% set hide_wrapper = True %}
+{% extends "case-study/base_case-study.html" %}
+
+{% block title %}Case Studies{% endblock %}
+{% block meta_description %}Ubuntu Case Studies{% endblock %}
+
+<!-- Set resource_types and tags somewhere -->
+{% block content %}
+<section class="p-strip--suru-blog-header is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1>Case Studies</h1>
+      <p>Ubuntu case studies.</p>
```

Comment:
I think this text is redundant here, since it almost duplicates the text above (h1)